### PR TITLE
 Implement additional ECMAScript Features for Web Browsers

### PR DESCRIPTION
**Pull request by ksh8281**

### CODE REVIEW

File: src/runtime/EnvironmentRecord.h [hunk 1]
```diff
@@ -572,6 +578,8 @@ class DeclarativeEnvironmentRecordNotIndexed : public DeclarativeEnvironmentReco
 
 private:
     bool m_isVarDeclarationTarget : 1;
+    bool m_isCatchClause : 1;
+    Optional<InterpretedCodeBlock::BlockInfo*> m_blockInfo;
```
Comment (clover2123) [inline src/runtime/EnvironmentRecord.h]: Is `m_blockInfo` necessary? Maybe I'm wrong but it seems that this member variable is nowhere used now.

**Comment by clover2123** [comment]
There is a conflict in `excludelist.orig.xml`. Please rebase this pr again.

File: src/runtime/EnvironmentRecord.h (hunk 1 above)
Comment (ksh8281) [reply src/runtime/EnvironmentRecord.h]: That is correct!

**Comment by ksh8281** [comment]
I resolved confilct

**Pull request merged**